### PR TITLE
Add missing reference types.

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

File: osv/vulnerability.proto [hunk 1]
```diff
@@ -152,13 +152,16 @@ message Credit {
 message Reference {
   enum Type {
     NONE = 0;
-    WEB = 1;
     ADVISORY = 2;
-    REPORT = 3;
-    FIX = 4;
-    PACKAGE = 5;
     ARTICLE = 6;
```
Comment (oliverchang) [inline osv/vulnerability.proto:156]: Also ordered these lexicographically

Comment (andrewpollock) [reply osv/vulnerability.proto:156]: Please add a comment with what the next ID is, given it's now much more difficult to determine this.

**Review by michaelkedar — APPROVED** [review]

**Review by andrewpollock — CHANGES_REQUESTED** [review]
To confirm, the only actual change of significance is to the osv/vulnerability.proto file?

**Comment by oliverchang** [comment]
> To confirm, the only actual change of significance is to the osv/vulnerability.proto file?

Yes, everything else is all auto-generated.

File: osv/vulnerability.proto (hunk 1 above)
Comment (oliverchang) [reply osv/vulnerability.proto:157]: done!

**Review by andrewpollock — APPROVED** [review]

**Pull request merged**